runtime-decompiler/src/plugins/ProcyonDecompilerWrapper.json

```diff
@@ -2,7 +2,9 @@
   "Name": "Procyon",
   "WrapperURL": "file:///etc/java-runtime-decompiler/plugins/ProcyonDecompilerWrapper.java",
   "DependencyURL": [
-    "file:///usr/share/java/procyon-decompiler.jar"
+    "file:///usr/share/java/procyon-decompiler.jar",
+    "file:///usr/share/java/procyon/procyon-core.jar",
+    "file:///usr/share/java/procyon/procyon-compilertools.jar"
   ],
   "DecompilerDownloadURL": "https://bitbucket.org/mstrobel/procyon/downloads/"
 }
```

Comment:
Both decompilers chnages are wrong from the upstream project perspectives.
I would actually say, that both shold be     "file://${HOME}/IdeaProjects/decompiler/whatever/somejars.jar"

As that is where you put them if you are oin windows, or on debian or so. 

You had changed them to be fedora like. If there is wish to have fedora configs, then I would vote for  ProcyonDecompilerWrapperFedora.java and FernflowerDecompilerWrapper.java

Reply:
It doesn't really matter where these files point to. They are not installed when you compile from source. We can't know where people put their binaries after they download them. 
I think using system paths is better in this case. Using "${HOME}/IdeaProjects/" makes much larger assumptions about the system it will be running on. (Why not downloads or desktop?)
Another advantage I see is that Fedora package won't need a patch file.